packages/datadog-plugin-cucumber/src/index.js

```diff
@@ -52,6 +52,10 @@ function createWrapRun (tracer, testEnvironmentMetadata, sourceRoot) {
           const promise = run.apply(this, arguments)
           promise.then(() => {
             setStatusFromResult(testSpan, this.getWorstStepResult(), TEST_STATUS)
+          }).finally(() => {
+            testSpan.context()._trace.started.forEach((span) => {
+              span.finish()
```

Comment:
this is in the other testing framework instrumentation but was not in this one. I found out when adding the http integration test

Reply:
Should this be moved to some common wrapper then instead of copying the logic? Especially since eventually there will be a better way to handle this which would otherwise require updating every plugin doing this.

Reply:
yeah that's a good point. I'll do it